src/data/ethereum/auraBalancerLpPools.json

```diff
@@ -556,5 +556,25 @@
         "decimals": "1e18"
       }
     ]
+  },
+  {
+    "name": "aura-ohm-dai",
+    "address": "0x76FCf0e8C7Ff37A47a799FA2cd4c13cDe0D981C9",
+    "vault": "0x17dEc2AF8018f2F940D34787399eA123Ff292963",
```

Comment:
This should be balancer vault, not beefy vault

Reply:
@MirthFutures oh, thanks ser, fixed, added `OHM` also to coingeckoCoins for fetching price since it doesn't have significant univ2 lp